src/arbitrary.rs

```diff
@@ -239,26 +239,89 @@ mod tests {
                 0xac, 0x19, 0x84, 0x29,
             ],
         },
+        TestVector {
```

Comment:
These test vectors are generated by zcash-test-vectors; I don't know why they weren't included before.

Reply:
They weren't included before because at the time this module was written, they didn't exist. The initial set of test vectors only had the first few; these were added in a subsequent refactor PR.